pulp_container/app/utils.py

```diff
@@ -127,7 +127,7 @@ def has_task_completed(dispatched_task):
         Throttled: If the task did not finish within a predefined timespan.
 
     """
-    for dummy in range(3):
+    for _ in range(wait_in_seconds):
```

Comment:
We use `_` a lot as an alias for gettext. I believe, despite it being "best practice" using `_` in that way is unsafe.

Reply:
I changed it back to `dummy`.